tests/release/release-failures.go

```diff
@@ -34,6 +38,16 @@ var _ = framework.ReleaseSuiteDescribe("test-release-service-failures", func() {
 			// Create the managed namespace
 			_, err = framework.CommonController.CreateTestNamespace(managedNamespace)
 			Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", managedNamespace, err)
+
+			// Wait until the "pipeline" SA is created and ready with secrets by the openshift-pipelines operator
+			klog.Infof("Wait until the %s SA is created in %s namespace \n", "pipeline", managedNamespace)
+			Eventually(func() bool {
+				sa, err := framework.CommonController.GetServiceAccount("pipeline", managedNamespace)
+				if sa == nil || err != nil || len(sa.Secrets) == 0 || len(sa.ImagePullSecrets) == 0 {
```

Comment:
Do you actually need to check for secrets and imagePullSecrets or this is just a nice-to-have?

Reply:
I realy do need to check it, because if I don't check the pipeline SA will not be ready yet (another race condition)

Reply:
If the SA is created, that stuff is already there as those are just references. I think you don't need it.